### DISCUSSION TITLE
Do dpkg needs more arguments than just -i package in termux?

### DISCUSSION THREAD

**Discussion by Inky1003** [termux/termux-packages#8234] [Q&A] [answered]
Hi, I wanted to install code-server 4.0.0 ".deb" package. As it can't be installed via yarn yet, I downloaded the deb and was about to install with dpkg.

Before that, I was thinking if installing with only `dpkg -i code-server.deb` would do anything wrong... do I need to put anything else as an argument to install the deb? Or it's safe to run just with -i ?

**Comment by sk0kanik** [comment, marked as answer]
Note : deb packages created for Linux cannot be installed normally because of [difference between Termux and Linux](https://wiki.termux.com/wiki/Differences_from_Linux).

Alternatively you can try [this](https://github.com/termux/termux-packages/discussions/7914#discussioncomment-1576420) or [this](https://coder.com/docs/code-server/latest/termux#installation).
